### ISSUE TITLE
Mechbay UI: Add an X to any removable component on a body part

### ISSUE THREAD

**Issue by dlikhten** [MechWarriorOnline/issue-tracker#222]
* [How To Upvote This Issue](https://github.com/MechWarriorOnline/issue-tracker/wiki/How-To-Upvote-An-Issue)
* [When To Add A Comment To An Issue (vs Just Upvoting)](https://github.com/MechWarriorOnline/issue-tracker/wiki/When-To-Add-A-Comment-To-An-Issue)

---

The Problem: Currently you have to drag components off body parts to somewhere not valid.

Proposal: This could be a much better experience by just adding an X to the right side of the component rectangle to for click-to-remove functionality.

**Comment by Tyler799** [comment]
I never even considered it, but that makes significantly more sense.

**Comment by tauSentry** [comment]
Administrivia...  
@dlikhten, can you add a mockup screenshot of how you envision this looking.

**Comment by Michal-Gawlas** [comment]
Double-clicking an item removes it without dragging.

**Comment by Tyler799** [comment]
I had no idea that was the case! Thanks! Should really be documented somewhere...

**Comment by tauSentry** [comment]
Administrivia...  
@dlikhten, does the feature pointed out by @TheFifthHorseman, address your need?

**Comment by dlikhten** [comment]
@tauSentry I think the X is important. The current functionality is great, but very unintuitive (I used this for 2 years and had no idea) I highly recommend the X is added and double-click is removed.

**Comment by Michal-Gawlas** [comment]
I agree that an explicit icon for the action should be added, but I don't think removing the double-click is a good idea. Way it's set up now, it allows you to very quickly remove an item without having to hunt at a specific part of it - and then do the same to the next item that appears in the slot.

**Comment by Ialdabaoth** [comment]
[https://www.dropbox.com/s/0zfy6yxgzurhccq/mechlab_mockup.png?dl=0](https://www.dropbox.com/s/0zfy6yxgzurhccq/mechlab_mockup.png?dl=0)
